core/renderers/zelos/info.js

```diff
@@ -14,23 +14,26 @@
 goog.module('Blockly.zelos.RenderInfo');
 goog.module.declareLegacyNamespace();
 
-goog.require('Blockly.blockRendering.InRowSpacer');
-goog.require('Blockly.blockRendering.RenderInfo');
-goog.require('Blockly.blockRendering.Types');
-goog.require('Blockly.constants');
-goog.require('Blockly.FieldImage');
-goog.require('Blockly.FieldLabel');
-goog.require('Blockly.FieldTextInput');
-goog.require('Blockly.inputTypes');
-goog.require('Blockly.utils.object');
-goog.require('Blockly.zelos.BottomRow');
-goog.require('Blockly.zelos.RightConnectionShape');
-goog.require('Blockly.zelos.TopRow');
-
-goog.requireType('Blockly.blockRendering.Measurable');
-goog.requireType('Blockly.BlockSvg');
-goog.requireType('Blockly.zelos.ConstantProvider');
-goog.requireType('Blockly.zelos.Renderer');
+const BaseRenderInfo = goog.require('Blockly.blockRendering.RenderInfo');
```

Comment:
Did we decide that this naming was ok?

Reply:
Maybe it would be better to create a `Blockly.baseRenderer` namespace which is separate from the `Blockly.blockRendering` namespace. But I guess that's a breaking change...

Reply:
Discussed offline, moving forward with this naming.